### PR TITLE
feat(cdp): Zendesk hog function

### PR DESCRIPTION
**Pull request by MarconLP**
## Change

- adds a zendesk hog function template

## How did you test this code?

added tests + manual testing

### CODE REVIEW

File: posthog/cdp/templates/__init__.py [hunk 1]
```diff
@@ -42,6 +43,7 @@
     avo,
     gleap,
     google_pubsub,
+    zendesk,
```
Comment (mariusandra) [inline posthog/cdp/templates/__init__.py:46]: we should probably sort this list alphabetically... too many conflicts with master on these lines 😅

File: posthog/cdp/templates/zendesk/template_zendesk.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        },
+        {
+            "key": "attributes",
+            "type": "dictionary",
+            "label": "Attribute mapping",
+            "description": "Map of Zendesk user fields and their values. You'll need to create User fields in Zendesk for these to work.",
+            "default": {
+                "phone": "{person.properties.phone}",
+                "plan": "{person.properties.plan}",
+            },
```
Comment (mariusandra) [inline posthog/cdp/templates/zendesk/template_zendesk.py:92]: Will these default fields work by default if users enable the function without thinking too much?

**Review by mariusandra — APPROVED** [review]

File: posthog/cdp/templates/__init__.py (hunk 1 above)
Comment (MarconLP) [reply posthog/cdp/templates/__init__.py:46]: 😅

File: posthog/cdp/templates/zendesk/template_zendesk.py (hunk 2 above)
Comment (MarconLP) [reply posthog/cdp/templates/zendesk/template_zendesk.py:92]: They'll have to create the `phone` and `plan` user fields in Zendesk beforehand, but Zendesk will just ignore those if not set.

**Pull request merged**